nepali_datetime/__init__.py

```diff
@@ -154,7 +154,8 @@ def _bin_search(key, *arr):
 
 def _check_tzname(name):
     if name is not None and not isinstance(name, str):
-        raise TypeError("tzinfo.tzname() must return None or string, not '%s'" % type(name))
+        e = """tzinfo.tzname() must return None or string, not '{}'""".format(type(name))
```

Comment:
@machinexa2 can you get rid of triple quotes & use f-string here as well. & no need to create new variable `e`

Reply:
I am currently sick. Will do it later